src/ansys/hps/client/client.py

```diff
@@ -190,6 +191,26 @@ def __init__(
                 verify=self.verify,
             )
             self.access_token = tokens["access_token"]
+
+            parsed_username = None
+
+            try:
+                parsed_jwt = jwt.decode(self.access_token, options={"verify_signature": False})
+                parsed_username = parsed_jwt["preferred_username"]
+            except:
+                log.warning("Could not retrieve preferred_username from access token.")
```

Comment:
```suggestion
                log.warning("Could not retrieve 'preferred_username' from access token.")
```
When parameter names must be used in message strings, we try to encourage surrounding them in single quotations.

Reply:
*Fixed.